engines/v8-debug/predict-file-name.js

```diff
@@ -15,24 +15,16 @@
 
 const predictFileName = (os) => {
 	switch (os) {
-		case 'mac64': {
-			return 'mac64';
-		}
-		case 'linux32': {
-			return 'linux32';
-		}
-		case 'linux64': {
-			return 'linux64';
-		}
-		case 'win32': {
-			return 'win32';
-		}
-		case 'win64': {
-			return 'win64';
-		}
+		case 'mac64':
+		case 'mac-arm64':
```

Comment:
The reason we had the explicit return values before was to differentiate between jsvu-exposed identifiers such as `mac64arm` and whatever the individual engines decide to use in their archive file names (e.g. `mac-arm64`)